src/Microsoft.AspNet.Authentication/AuthenticateResult.cs

```diff
@@ -44,6 +46,11 @@ public static AuthenticateResult Success(AuthenticationTicket ticket)
             return new AuthenticateResult() { Ticket = ticket };
         }
 
+        public static AuthenticateResult Skip()
+        {
+            return new AuthenticateResult() { Skipped = true };
+        }
+
         public static AuthenticateResult Failed(Exception error)
```

Comment:
Should `Failed` be renamed to `Fail` to match `Skip`'s form? (FYI: https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server/issues/173#issuecomment-155238696)